hathor/cli/util.py

```diff
@@ -139,11 +156,15 @@ def __call__(self, _, __, event_dict):
                 # },
             },
             'loggers': {
+                'twisted': {
+                    'handlers': ['default'],
+                    'level': 'INFO' if debug else 'WARN',
+                    'propagate': False,
+                },
                 '': {
-                    # 'handlers': ['default', 'file'],
                     'handlers': ['default'],
                     'level': 'DEBUG' if debug else 'INFO',
-                    'propagate': True,
+                    # 'propagate': True,
```

Comment:
Should we remove this line? Or add a comment explaining when it should be uncommented?

Reply:
I'll link to python's logging docs, which explain what all theses settings mean and remove that line.

Reply:
Done. What do you think?